nixos/modules/hardware/fwenv.nix

```
@@ -0,0 +1,65 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.fwenv;
+in
+
+{
+  options.hardware.fwenv = {
+    enable = lib.mkEnableOption (lib.mdDoc ''
+      U-Boot environment access configuration.
+
+      Use this to setup access to the U-Boot's environment variables with
+      generic ubootEnvTools package.
+    '');
+    package = lib.mkOption {
```

Add some spaces after `;`s, like

```suggestion

    package = lib.mkOption {
```